icubam/messaging/handlers/onoff.py

```diff
@@ -0,0 +1,50 @@
+from absl import logging
+import dataclasses
+import json
+import tornado.web
+
+
+@dataclasses.dataclass
+class OnOffRequest:
+  user_id: int = None
+  icu_id: int = None
+  on: bool = True
+  delay: int = None
+
+  def to_json(self):
+    return json.dumps(dataclasses.asdict(self))
+
+  def from_json(self, encoded):
+    self.__init__(**json.loads(encoded))
+
+
+class OnOffHandler(tornado.web.RequestHandler):
+
```

Comment:
Can you add a comment about the purpose of the handler? It will help if one doesn't know the context,